cmd/emp/dynos.go

```diff
@@ -58,6 +58,9 @@ func listDynos(w io.Writer) {
 func listDyno(w io.Writer, d *heroku.Dyno) {
 	listRec(w,
 		d.Name,
+		d.Id,
+		d.ContainerInstanceID,
```

Comment:
In general, the container instance id isn't very useful, and it's easy to correlate in AWS if we have the EC2 instance id. I think we should leave this off for the first version.